mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -601,12 +599,13 @@ public void onItemClick(AdapterView<?> parent, View view, int index, long id) {
 						Method method = Spinner.class.getDeclaredMethod("onDetachedFromWindow");
 						method.setAccessible(true);
 						method.invoke(selectionSpinner);
-					} catch (Exception ex) {}
+					} catch (Exception ignored) {}
 				}
 			});
-    		spinnerAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-    		selectionSpinner.setAdapter(spinnerAdapter);
-    		if (spinnerSelectedIndex >= 0) {
+			spinnerAdapter.setDropDownViewResource(R.layout.openhabwidgetlist_sectionswitchitem_spinner);
+			selectionSpinner.setAdapter(spinnerAdapter);
+			selectionSpinner.setPrompt(openHABWidget.getLabel());
+            if (spinnerSelectedIndex >= 0) {
```

Comment:
you've mixed whitespaces with tabs :)

Reply:
I noticed that, but I couldnt find a way to use spaces in android studio, so I opened the file in vim and saw that the whole file is mixed.

Reply:
We should write down rules for text indetion. Four spaces?